Formula/lib/libpostal-rest.rb

```diff
@@ -0,0 +1,54 @@
+class LibpostalRest < Formula
+  desc "REST API for libpostal"
+  homepage "https://github.com/johnlonganecker/libpostal-rest"
+  url "https://github.com/johnlonganecker/libpostal-rest/archive/refs/tags/v1.1.0.tar.gz"
+  sha256 "d02d738fe1d8aee034c47ff9e8123e55885fe481f1a6307fbfe286b7b755468d"
+  license "MIT"
+
+  depends_on "go" => :build
+  depends_on "pkgconf" => :build
+  depends_on "libpostal"
+
+  def install
+    system "go", "build", *std_go_args(ldflags: "-s -w")
+  end
+
+  test do
+    require "json"
+
+    port = free_port
+    pid = fork do
+      ENV["LISTEN_PORT"] = port.to_s
+      exec bin/"libpostal-rest"
+    end
+    sleep 15
```

Comment:
is 15 the same for both arm and intel runners? 

you can also probably do

sleep 5
sleep 10 if OS.mac? && Hardware::CPU.intel?